helper/linker.go

```diff
@@ -62,6 +62,10 @@ func (f FileLinker) Execute() (map[string]string, error) {
 		layerDir = "/layers/paketo-buildpacks_open-liberty/open-liberty-runtime"
 	}
 
+	if _, err := os.Stat(layerDir); os.IsNotExist(err) {
+		return map[string]string{}, fmt.Errorf("runtime root '%v' does not exist", layerDir)
+	}
+
```

Comment:
```suggestion
        _, err := os.Stat(layerDir)
	if err != nil && os.IsNotExist(err) {
		return map[string]string{}, fmt.Errorf("unable to find '%s', folder does not exist", layerDir)
	} else if err != nil {
	        return map[string]string{}, fmt.Errorf("unable to check %s\n%w", layerDir, err)
	}
```

I think you're missing an error check here.